packages/nimble-components/src/dialog/tests/dialog.spec.ts

```diff
@@ -1,14 +1,15 @@
 import { html } from '@microsoft/fast-element';
 import { fixture, Fixture } from '../../utilities/tests/fixture';
-import { Dialog, dialogTag, ExtendedDialog, UserDismissed } from '..';
+import { Dialog, dialogTag, UserDismissed } from '..';
 import { waitForUpdatesAsync } from '../../testing/async-helpers';
+import type { ExtendedDialog } from '../../dialog-base';
 
 // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
 async function setup<CloseReason = void>(
     preventDismiss = false
 ): Promise<Fixture<Dialog<CloseReason>>> {
     const viewTemplate = html`
-        <nimble-dialog ?prevent-dismiss="${() => preventDismiss}">
+        <nimble-dialog ?prevent-dismiss="${() => preventDismiss ?? null}">
```

Comment:
Why do you need the `?? null` here? Isn't the type of `preventDismiss` a `boolean`, which would mean it is never not defined?

Reply:
You're right. I must have been confused.